src/MIDebugEngine/AD7.Impl/AD7PendingBreakpoint.cs

```diff
@@ -113,8 +115,9 @@ private bool CanBind()
             }
             if ((_bpRequestInfo.dwFields & enum_BPREQI_FIELDS.BPREQI_PASSCOUNT) != 0)
             {
-                this.SetError(new AD7ErrorBreakpoint(this, ResourceStrings.UnsupportedPassCountBreakpoint, enum_BP_ERROR_TYPE.BPET_GENERAL_ERROR));
-                return false;
+                hitCount = _bpRequestInfo.bpPassCount.dwPassCount;
+                //this.SetError(new AD7ErrorBreakpoint(this, ResourceStrings.UnsupportedPassCountBreakpoint, enum_BP_ERROR_TYPE.BPET_GENERAL_ERROR));
```

Comment:
>SetError [](start = 23, length = 8)

If we are going to support hit count breakponits, don't we need to tell GDB about the hit count and reset the hit count in GDB if needed?